src/SFA.DAS.Forecasting.Domain/Estimations/AccountEstimationProjection.cs

```diff
@@ -55,18 +55,29 @@ private AccountProjectionModel CreateProjection(DateTime period, decimal lastBal
         {
             var totalCostOfTraning = _virtualEmployerCommitments.GetTotalCostOfTraining(period);
             var completionPayments = _virtualEmployerCommitments.GetTotalCompletionPayments(period);
-            var commitments = totalCostOfTraning.Item2;
-            commitments.AddRange(completionPayments.Item2);
-            commitments = commitments.Distinct().ToList();
+
+            var commitments =
+                totalCostOfTraning.CommitmentIds
+                .Concat(completionPayments.CommitmentIds)
+                .Distinct();
+
+            var costOfTraining = totalCostOfTraning.LevyFunded + totalCostOfTraning.TransferOut;
+
             var balance = lastBalance;
             var projection = new AccountProjectionModel
             {
                 FundsIn = _account.LevyDeclared,
                 EmployerAccountId = _account.EmployerAccountId,
                 Month = (short)period.Month,
                 Year = (short)period.Year,
-                TotalCostOfTraining = totalCostOfTraning.Item1,
-                CompletionPayments = completionPayments.Item1,
+                TotalCostOfTraining = costOfTraining,
```

Comment:
TotalCostOfTraining stores the cost of training for an employer's own commitments.  This should only store totalCostOfTrainingLevyFunded as this allows us to show a detalled breakdown of costs in a later version of the UI.